backend/upload.js

```diff
@@ -2,35 +2,56 @@ import multer from 'multer'
 import mkdirp from 'mkdirp'
 import uuidv1 from 'uuid/v1'
 
+const test_dir = './test/images'
+const image_dir = './images'
+
 const Storage = multer.diskStorage({
   destination: (req, file, callback) => {
-    callback(null, './images') // TODO: See if we want a ./year/images
+    if (process.env.NODE_ENV === 'test'){
+      callback(null, test_dir) // TODO: See if we want a ./year/images
+    } else {
+      callback(null, image_dir) // TODO: See if we want a ./year/images
+    }
   },
   filename: (req, file, callback) => {
     // Generate a unique id for the file
     const filename = uuidv1()
```

Comment:
Now that I see this ... should we be using v1 here? Is there any reason to avoid v4? Hmm, will discuss later, not a problem for now